src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -905,29 +905,60 @@ namespace cryptonote
       return 1;
     }
 
-    std::vector<cryptonote::blobdata> newtxs;
-    newtxs.reserve(arg.txs.size());
-    for (size_t i = 0; i < arg.txs.size(); ++i)
+    relay_method tx_relay;
+    std::vector<blobdata> stem_txs{};
+    std::vector<blobdata> fluff_txs{};
+    if (arg.dandelionpp_fluff)
     {
-      cryptonote::tx_verification_context tvc{};
-      m_core.handle_incoming_tx({arg.txs[i], crypto::null_hash}, tvc, relay_method::fluff, true);
-      if(tvc.m_verifivation_failed)
+      tx_relay = relay_method::fluff;
+      stem_txs.reserve(arg.txs.size());
+    }
+    else
+    {
+      tx_relay = relay_method::stem;
+      fluff_txs.reserve(arg.txs.size());
+    }
```

Comment:
Are the reserved vectors swapped w.r.t arg.dandelionpp_fluff ?

Reply:
Nope, I botched this. Initially the field was called `dandelionpp_stem`, but I flipped the logic so that zero initialization of the `struct` would default to stem mode. And since this can't be easily observed through testing ... anyway will change.